CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,31 @@
+# NoOps Code of Conduct v1.0 #
+
+[日本語版](#日本語版) below
+
+# Introduction
```

Comment:
It's better to be H2 instead of H1. Likewise, `Expected Behavior`, `Unacceptable Behavior`, `Scope`, `Consequences` as well.

Reply:
if we have a subtitle (h2) then we should probably add a main title (e.g. NoOps Code of Conduct) as the main header?

Reply:
downgraded formatting